zigpy/zcl/clusters/greenpower.py

```diff
@@ -18,7 +18,17 @@
 
 # Figure 27
 class CommissioningNotificationSchema(foundation.CommandSchema):
```

Comment:
Went through the tests today and found this convenient IntStruct type, which I've migrated to instead of properties. Question though: is the `_reserved` padding field required, or can it be safely omitted?